tests/impl.cpp

```diff
@@ -3117,7 +3117,22 @@ result_t test_mm_blendv_epi8(const SSE2NEONTestImpl &impl, uint32_t i)
 
 result_t test_mm_slli_epi32(const SSE2NEONTestImpl &impl, uint32_t i)
 {
-    return TEST_UNIMPL;
+    const int32_t *_a = (const int32_t *) impl.mTestIntPointer1;
+    const int32_t *_b = (const int32_t *) impl.mTestIntPointer2;
+#if defined(__clang__)
+    int count = (uint32_t) _b[0] % 32;
```

Comment:
Why don't you use `i` instead of `_b[0]`?
I am just curious.

Reply:
I think `_b[0]` is more random value and `i` is a simple increasing `int`. Therefore, I choose the radnom one.